modules/administration/pages/disconnected-setup.adoc

```diff
@@ -105,7 +105,8 @@ For more details on installing and configuring SMT, see https://documentation.su
 smt-sync --createdbreplacementfile /tmp/dbrepl.xml
 ----
 . Export the synchronized data to your removable storage.
-    In this example, the storage medium is mounted at [path]``/mnt/usb``:
+//    In this example, the storage medium is mounted at [path]``/mnt/usb``:
+    TO BE CHECKED WITH DEV: In this example, the storage medium is mounted in the containerized {productname} Server on [path]``/mnt/usb``:
```

Comment:
This part is to be executed on a different machine that has access to the repos and internet, so no Uyuni container involved at that point.

The server container should be installed with the `mgradm install podman --mirrorPath /media/disk` rather than adding `server.susemanager.fromdir = /media/disk` to `/etc/rhn/rhn.conf`. At the moment we have no way to change this option after the installation.

The `systemctl` and `mgr-sync` commands in the last procedure should be prepended with `mgrctl exec --  ` to be executed in the container.